### PR TITLE
HPCC-13272 Fix to allow localThor setup to work with hostnames

### PR DESCRIPTION
**Pull request by jakesmith**
With a local Thor setup (localThor=true), if hostnames were used
in the environmnent as opposed to "." or IP's, Thor failed to
start for a couple of reasons.
a) the daliadmin command lines in the script expected "." or IP's.
Fixed with addition of server=
b) the hostname failed to match the IP's in the thorgroup with
the consequence that no slaves were started by the start_slaves
scripts. Fixed by using an IP from the group.

Signed-off-by: Jake Smith jake.smith@lexisnexis.com

### CODE REVIEW

**Comment by jakesmith** [comment]
@Michael-Gardner - please check

**Comment by Michael-Gardner** [comment]
Works well and I see no issues with the code.

**Comment by Michael-Gardner** [comment]
Current commit has compilation errors and -DCASS_USE_EXTERNAL_BOOST=ON is required to build on my machine.  It might be an error on my side but I'm building fresh candidate-5.4.0 branches without issue.

File: plugins/cassandra/cpp-driver [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 189e46e3386cc2be0fd7428c1072339b9c611d6e
+Subproject commit 019d3f62b39eb6ebf0a8605faa7130bd9863a9f9
```
Comment (richardkchapman) [inline plugins/cassandra/cpp-driver]: This change seems unrelated and is probably responsible for the CASS_USE_BOOST issues.

**Comment by AttilaVamos** [comment]
Automated Smoketest Build failed

**Comment by jakesmith** [comment]
There was a stray unrelated commit, that wasn't supposed to be part of this branch.
@Michael-Gardner - please look again.

**Comment by Michael-Gardner** [comment]
Works perfectly
@richardkchapman
